Automattic/pocket-casts-android
# Update Top 1 Show story texts

## Description

Part of feedback from #3169

> In this screen I believe we made a change to add the title of the show to the paragraph instead of the heading so it scales better

## Testing Instructions

1. Start PB24.
2. Go to Top 1 Show story.
3. Verify text copies.

## Screenshots or Screencast 

| Before | After |
| - | - |
| ![Screenshot 2024-11-05 at 09 59 33](https://github.com/user-attachments/assets/229f10f5-c0de-4e80-bf4c-c2e5a99774fc) | ![Screenshot 2024-11-05 at 09 56 33](https://github.com/user-attachments/assets/1bf90d20-571b-48ae-a387-ee4eee7113aa) |

## Checklist
